logstreams/src/main/java/io/zeebe/logstreams/impl/log/LogStreamImpl.java

```diff
@@ -335,13 +324,13 @@ private int determineInitialPartitionId() {
       final long lastPosition = logReader.seekToEnd();
 
       // dispatcher needs to generate positions greater than the last position
-      int partitionId = 0;
+      int dispatcherPartitionId = 0;
```

Comment:
This could cause merge conflicts - I think I renamed it already :D

Reply:
:see_no_evil: